test/test_fx_experimental.py

```diff
@@ -1199,6 +1199,15 @@ def forward(self, inp: Tuple[CustomType, torch.Tensor], inp2: List[CustomType],
                 inp3_y = inp3.y
                 return inp_0 + inp_1 + inp2_0 + inp3_x + inp3_y
 
+        class MyModule2(torch.nn.Module):
```

Comment:
Is the content of this file a bad merge?

Reply:
No, MyModule2 was added to test the new syntax of type annotations.